CONTRIBUTING.MD

```diff
@@ -0,0 +1,35 @@
+## Contributing
+
+You are more than welcome to make any contributions.
+Please create Pull Request for any changes.
+
+### Fork the Repository
+
+* When contributing to the library it is recommended that you fork the repository into you own. See [Github's Fork Guide](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) for more information
+
+* Create a branch from 'master' and implement your changes
+
+* Raise a pull request(PR) back to the master branch of gocelery
+
+* It is good practice to reference an issue number, or explain what your enhancement lends to the project in the PR comments
+
+* Continuous Integration builds must pass on the PR
+
+* Newly implemented features or bug fixes must be accompanied by tests
+
+* To get the latest changes from the forked repository [setup the upstream master](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) and [sync](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork) often 
+
+### Installing for Local Development
```

Comment:
I think we should document integration testing process instead of a local build. You can describe a process of running `example/goworker/main.go` and `example/goclient/main.go` to ensure things are working properly. Also test section should precede this section.